assets/js/googlesitekit/data/create-fetch-store.js

```diff
@@ -148,6 +149,12 @@ export const createFetchStore = ( {
 				type: START_FETCH,
 			};
 
+			const registry = yield Data.commonActions.getRegistry();
+			const { clearError } = registry ? registry.dispatch( storeName ) : {};
+			if ( clearError ) {
+				yield clearError( baseName, params );
+			}
```

Comment:
@felixarntz Gutenberg core uses generic controls for dispatching actions from other actions, which is a bit cleaner as it avoids the need for the `getRegistry` or a dedicated control for simple things.

We don't actually need the registry for these actions though, we only need access to the "raw" unbound actions - `registry.dispatch()` returns bound action creators that are dispatched onto the given store when called. When we yield a "raw" action, it will be dispatched onto the current store automatically which makes `storeName` unnecessary. If the store does not "implement" the error store, then the action is simply ignored.

I've gone ahead and updated this in my PR for #1801 and we can iterate on the approach there if necessary.